tests/create_users_policies/main.tf

```
@@ -168,7 +164,7 @@ locals {
     account_id    = data.aws_caller_identity.current.account_id
     partition     = data.aws_partition.current.partition
     region        = data.aws_region.current.name
-    random_string = random_string.this.result
+    random_string = "${random_string.this.result}"
```

This interpolation is not necessary. Somewhat surprised `terraform fmt -recursive` is not removing it.

```suggestion
    random_string = random_string.this.result
```

---

ok